ament_tools/build_types/cmake.py

```diff
@@ -221,12 +221,12 @@ def _common_cmake_on_build(self, should_run_configure, context, prefix, extra_cm
                 cmd += msbuild_flags
                 # If there is a parallelism flag in msbuild_flags and it's not /m1,
                 # then turn on /MP for the compiler (intra-project parallelism)
-                if any([x.startswith('/m') for x in msbuild_flags]) and \
+                if any(x.startswith('/m') for x in msbuild_flags) and \
```

Comment:
Is this really recommended? It seems to provide less information about what is happening and requires more expert knowledge to know that `any` is capable of this. -1 from me.

Reply:
Yeah I agree, this one is the only one that made me hesitate.
I think it's clearer that we have a list here if keeping the `[`